### PR TITLE
Fix HttpLoggingInterceptor crash

### PR DESCRIPTION
**Pull request by adasiewiczr**
### 🎯 Goal

Fix `HttpLoggingInterceptor` crash.

### 🛠 Implementation details

Added a limit for writing/reading from the buffer. Now we are not going to log request bodies of more than 1MB.

### 🧪 Testing

Send a big attachment (> 30MB).

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] Bugs validated (bugfixes)
- [ ] All code we touched has new or updated KDocs

### CODE REVIEW

File: stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Constants.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * Number of bytes in a megabyte.
+     */
+    public const val MB_IN_BYTES: Long = 1024 * 1024
+
+    /**
+     * Maximum request body length in bytes.
+     */
+    internal const val MAX_REQUEST_BODY_LENGTH = 1 * MB_IN_BYTES
+}
```
Comment (JcMinarro) [inline stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Constants.kt:33]: Do we need it public?

**Review by JcMinarro — APPROVED** [review]
It looks good.
I only left a comment about visibility of the new class

File: stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Constants.kt (hunk 1 above)
Comment (adasiewiczr) [reply stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Constants.kt:33]: good catch. Marking with @InternalStreamChatApi

**Review by JcMinarro — APPROVED** [review]

**Review by bychkovdmitry — APPROVED** [review]
Looks good from the UI side!

**Pull request merged**